public/app/features/explore/QueryRow.tsx

```diff
@@ -138,6 +138,7 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
           onBlur={noopOnBlur}
           onChange={this.onChange}
           data={queryResponse}
+          range={range}
           absoluteRange={absoluteRange}
```

Comment:
In Explore, we were passing range to Angular editors, now we are passing it everywhere.